src/chrome/content/rules/Gtimg.com.xml

```diff
@@ -8,20 +8,25 @@
         <target host="a.map.gtimg.com" />
                         -->
 
-        <!--    mixed content
-        <target host="ra.gtimg.com" />
-        <test url="https://ra.gtimg.com/icast/1452425/1452425.htm" />
-        <test url="http://ra.gtimg.com/web/default_fodders/300x250_1.swf" />
-                        -->
-
-        <target host="cdnjs.gtimg.com" />
         <target host="static.gtimg.com" />
+        
+        <target host="cdnjs.gtimg.com" />
         <target host="wximg.gtimg.com" />
-
         <!--    Test them at:
         https://wximg.gtimg.com/tmt/_events/promo/EyxiHkkq/html/index.html
         -->
         
+        <target host="ra.gtimg.com" />
+                <test url="http://ra.gtimg.com/sc/vqq/crystal-min.js" />
+                <test url="http://ra.gtimg.com/web/default_fodders/qqliveweb/300x250_1.swf?v=20130124" />
+
+        <exclusion pattern="^http:\/\/ra\.gtimg\.com\/\w+\/\d+\/\d+\.html?$" />
```

Comment:
Please document that you made this exclusion because of mixed content errors.
Slashes don't need to be escaped. Removing the backslashes does also improve readability.